Settings.h

```diff
@@ -34,15 +37,17 @@ typedef struct {
 } MeterColumnSetting;
 
 typedef struct {
```

Comment:
Shouldn't this be called like this?
```suggestion
typedef struct ScreenSettings_ {
```

Reply:
I think it depends on whether any other code might want to use the struct without pulling in the header, but sure, I'll update it - may as well be same as struct Settings here (though ScreenDefaults and MeterColumnSetting in the same file don't use this technique - *shrug*)